app/index.html

```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <title>Ember.js - Deprecations</title>
     <meta name="description" content="">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1.0, user-scalable=0"">
```

Comment:
```suggestion
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=yes"">
```
`user-scalable` has to be `yes` for accessibility reasons.